packages/debs/SPECS/wazuh-agent/debian/postinst

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+# postinst script for wazuh-agent
+# Wazuh, Inc 2024
+
+set -ex
```

Comment:
We could consider removing the -x option now, or leave a TODO comment to remove it once we have a final version ready. Adding a TODO would help us keep track of the change for later refinement.

Reply:
The same applies to the other scripts.